src/Invictus.Testing/Invictus.Testing.csproj

```diff
@@ -17,8 +17,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Arcus.Testing.Logging" Version="0.1.0" />
     <PackageReference Include="Guard.NET" Version="1.2.0" />
     <PackageReference Include="Microsoft.Azure.Management.Logic" Version="4.1.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
```

Comment:
Used this package to have access to the `IAsyncDisposable`, ... stuff, but when we would upgrade to `netstandard2.1`, we wouldn't need that. Just wasn't sure if that (`netstandard2.1`) was something we would have to force on consumers or not.

Reply:
We could make use of the `IAsyncEnumerable` and all instead of several `Task<IEnumerable<>>` that are now written here.

Reply:
Currently, not a lot of customers are actually using this framework yet, so it would be better to change it earlier on in the process instead of doing it for a future version.

This will not impact the project of the Logic Apps-integration in itself, as those are only ARM-templates, not actual code, so no problems there.